tests/manage/z_cluster/nodes/test_automated_recovery_from_failed_nodes_proactive_IPI.py

```diff
@@ -155,6 +155,8 @@ def test_automated_recovery_from_failed_nodes_IPI_proactive(
                         command = f"delete deployment {deployment_name}"
                         ocp_obj.exec_oc_cmd(command=command)
                         log.info(f"Deleted deployment for pod {pod_obj.name}")
+                    else:
```

Comment:
Hi @PrasadDesala , please review if this change is ok with this test case you created